cmd/podman/pod_stop.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	podStopFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "stop all running pods",
+		},
+		cli.BoolFlag{
```

Comment:
When would I not want to do this?

Reply:
Not sure, I didn't really understand what it did. I can remove the option if that's preferred.

Reply:
Well if you don't know what cleanup option is for, then remove it.

Reply:
Hardcode it to true for podman